luigi/parameter.py

```diff
@@ -176,7 +187,16 @@ def _value_iterator(self, task_name, param_name):
 
         The parameter value will be whatever non-_no_value that is yielded first.
         """
-        yield (self.__global, None)
+        from luigi.cmdline_parser import CmdlineParser
+        cp_parser = CmdlineParser.get_instance()
+        if cp_parser:
```

Comment:
@erikbern, as you hoped for in #1197, it's not hard to not force an `CmdlinePaser` instance to be present. :)